index.html

```diff
@@ -5,45 +5,123 @@
             If you like what you see and have a project you need coded, 
             don’t hesitate and contact me.
         </p>
         <a href="#" class="collab" id="principal-link">Start collaboration</a>
     </div>
 
+    <img src="images/image-geometry_1.svg" alt="" class="image-big" />
+
     <img src="images/image_geometry_2.svg" alt="" class="image-down" />
 
+    <img src="images/image_dot.png" alt="" class="image-dot" />
+
     <section class="projects">
         <h2>Projects</h2>
         <div class="box one">
-            <h3>Project name goes here</h3>
+            <h3 class="first-title">Project name goes here</h3>
             <img class="component" src="images/component_1.png" alt="website" />
+
             <ul class="keywords">
                 <li>HTML/CSS</li>
                 <li>Ruby on Rails</li>
                 <li>Javascript</li>
             </ul>
             <button class="buttonbox">See this project<img src="images/Union.png"/></button>
         </div>
+
+        <!--FIRST CARD MOBILE-->
+
+        <div class="first">
+            <h3 class="first-title">Project name goes here</h3>
+            <img class="component-1" src="images/component_1_.png" alt="website" />
+
+            
+
+            <ul class="keywords-first">
+                <li>HTML/CSS</li>
+                <li>Ruby on Rails</li>
+                <li>Javascript</li>
+            </ul>
+            <button class="buttonbox-first">See this project<img src="images/Union.png"/></button>
+        </div>
+
+        <img class="dot-projects" src="images/image_dot.png" alt="" class="image-dot" />
+
+        <!--SECOND CARD MOBILE-->
+        <div class="second">
+            <h3 class="second-title">Project name goes...</h3>
+            <ul class="keywords-second">
+                <li>HTML/CSS</li>
+                <li>Ruby on Rails</li>
+                <li>+1</li>
+            </ul>
+            <button class="buttonbox-second">See this project<img src="images/Union.png"/></button>
+            <img class="component2" src="images/component_5.png" alt="website" />
+        </div>
+
+        <!--CENTER GEOMETRY IMG-->
+        <div class="geometry-center">
+            <img src="images/image-geometry_2.png" alt="">
+        </div>  
+        
+        <div class="pc-left">
+            <img src="images/pc_img_left.png" alt="">
+        </div>
+
+        <div class="pc-left-bigger">
+            <img src="images/pc_img_left_2.png" alt="">
+        </div>
+
+        <div class="pc-right-squares">
+            <img src="images/image-geometry_3.png" alt="">
+        </div>
+
+        <div class="dot-right-yellow">
+            <img src="images/image-geometry_4.png" alt="">
+        </div>
+
+        <div class="pc-left-squares">
+            <img src="images/image-geometry_5.png" alt="">
+        </div>
+
+        <div class="pc-right-yellow">
+            <img src="images/pc_right.png" alt="">
+        </div>
+
+        
```

Comment:
- [ ] Please use a consistent indentation of two spaces throughout your document. You can use an extension like prettier on vscode to format your document to fix all the indentation.
An indentation of two spaces will look like
```html
<div class="second">
  <h3 class="second-title">Project name goes...</h3>
    <ul class="keywords-second">
      <li>HTML/CSS</li>
      <li>Ruby on Rails</li>
      <li>+1</li>
    </ul>
    <button class="buttonbox-second">See this project<img src="images/Union.png"/></button>
    <img class="component2" src="images/component_5.png" alt="website" />
</div>
```

Reply:
@Kamba56 Hi! Thanks for your help! I think I don't understand the indentation change that you want. Can you give me a more detailed explanation?

Reply:
HELLO @MarcoDDM. Kindly check [here](https://www.learnhowtoprogram.com/introduction-to-programming-part-time-react-track/git-html-and-css/html-indentation-and-spacing) for more information. Indentation is simply the size of your tab. The space before every line of code.

Reply:
The spaces should be multiples of 2 depending on the nesting. I can see that yours is a multiple of 4.

Reply:
Hey @MarcoDDM , call me on zoom if you need help. I tried searching for you and I couldn't find you. Just search for `Toma John Musa`